src/components/Article.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropsTypes from 'prop-types';
+
+const Article = ({ title, text, date }) => (
+  <article>
+    <h1>{title}</h1>
+    <p>{date}</p>
+    <span>{text}</span>
+  </article>
+);
+
+Article.propsTypes = {
+  title: PropsTypes.string,
+  date: PropsTypes.string,
+  text: PropsTypes.string,
+};
```

Comment:
if values are not required, then you'd better specify default props

Reply:
added 'isRequired'